airbyte-webapp/src/components/connectorBuilder/StreamTestingPanel/StreamTestingPanel.module.scss

```diff
@@ -23,7 +23,6 @@ $buttonHeight: 36px;
 }
 
 .selectAndTestContainer {
```

Comment:
```suggestion
.selectAndTestContainer {
  height: 100%;
```

Reply:
I don't think this `height` style should have been removed. This messed up the sizes of the testing panel containers:
<img width="646" alt="Screen Shot 2023-01-03 at 4 23 28 PM" src="https://user-images.githubusercontent.com/22731524/210463012-085152ab-b042-4697-b8a2-1f5c6ed4c6f9.png">